### DISCUSSION TITLE
需要一名小提琴老师，要求：耐推即可。

### DISCUSSION THREAD

**Discussion by 5cheng7** [share121/inter-knot#2656] [其他]
![IMG_20240818_122114](https://github.com/user-attachments/assets/abfb5ea8-24d9-40c4-b5a0-55c201d4102a)

**Comment by cpu644** [comment]
我把电吉他架脖子上拿个弓行不行

**Comment by wwwsheepwww** [comment]
我居然会在这里看到omori的地狱笑话hh

**Comment by lllliufeng** [comment]
推荐传奇小提琴手阿乌拉（

**Comment by homteam** [comment]
🤓👆
